freedata_gui/src/js/eventHandler.js

```diff
@@ -153,6 +153,36 @@ export function eventDispatcher(data) {
       loadAllData();
       return;
 
+    case "message-logging":
+      if (data.endpoint === "wavelog") {
+        if (data.status === true) {
+            const message = `
+            <div>
+                <strong>${i18next.t('Wavelog posted')}:</strong>
```

Comment:
The strings should be translatable, so we need a reference in translation files.